src/main/java/ch/bfh/bti7081/s2018/green/presenters/AddEventPresenter.java

```diff
@@ -34,11 +34,19 @@ public AddEventPresenter(AddEventView view) {
 
 	// Update DateTimeFields without impacting Time when changing Dates
 	private void SetToDate(LocalDateTime from, LocalDateTime to) {
-		this.view.getDtfTo().setValue(to.withDayOfYear(from.getDayOfYear()));
+		try {
+			this.view.getDtfTo().setValue(to.withDayOfYear(from.getDayOfYear()));
+		} catch (NullPointerException npe) {
+			ErrorView.showError("From-Datefield format is invalid, please re-check", Page.getCurrent());   
```

Comment:
Do we really need to show an error? The datefield is clearly marked by Vaadin

Reply:
1. If i don't do it with try and catch, an ugly NPE appears that is visible for the user.
2. You created an issue on this check to implement error handling.

So, yes, i think we should show an error there.

Reply:
I created an issue to add exception handling. Since the user is already notified by Vaadin, I don't think it is needed to show another error. In this case I think ignoring the error is ok.

But whatever, we do not have any time left. Please fix the conflicts and I will merge as is